colombia.html

```diff
@@ -0,0 +1,313 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Colombia</title>
+    <link rel="stylesheet" href="/css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css">
+    <link rel="icon" href="/img/colombia/colombia.png">
+</head>
+<body>
+    <!--NAV-->
+    <nav>
+        <ul>
+            <li><a href="#place">Places</a></li>
+            <li><a href="#festival">Festival</a></li>
+            <li><a href="#food">Gastronomy</a></li>
+            <li><a href="#travel">Travel</a></li>
+        </ul>
+        <aside>
+            <i class="fa-solid fa-earth-americas"></i>
+        </aside>
+        <ul>
+            <li><a href="home.html">HOME</a></li>
```

Comment:
You have to change it to 'href='index.html'

Reply:
fixed!